flashing/flash_portapack_mayhem.bat

```diff
@@ -10,6 +10,17 @@ echo.
 pause
 
 echo.
+
+REM Check if the firmware file exists

```

Comment:
What's the workflow here? Clone the repo to get this script, but also copy the release stuff into this folder? Or does this script end up in the release in which case, shouldn't the image be there?

Reply:
If the flashing script can't find the firmware file, it tells the user where to download it from, as some users just download the files directly from the repo which doesn't include the firmware instead of the release which has it packaged with it